### PR TITLE
CAMP-24

### PR DESCRIPTION
**Pull request by hariwij**
- Add section about MongoDB Compass to the acedemy docs.

### CODE REVIEW

**Review by kesaralive — APPROVED** [review]

File: docs/content/services/connect_mongodb_database_cluster.md [hunk 1]
```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
...
+    </figure>
+    <sub>After running the downloaded file, you will see the installation wizard. Wait until the installation is completed.</sub>
+
+    -   After the installation is completed, it will automatically open the MongoDB Compass. If not, you can open it from the start menu. You can see the following screen.
+    <figure markdown>
+      <kbd>![compass-first-view.png](../../img/services/connect_mongodb_database_cluster/compass-first-view.png)</kbd>
+    </figure>
+        Close the welcome screen and click on **New connection**.
+
+    -   Then you need to provide the connection details. You can find the connection details in the **Connect** tab of your database cluster. You can see the following screen. Copy the connection URI and paste it in the **Connection String Only** field in MongoDB Compass. Then click on **Save and Connect**.
```
Comment (IsuruVihan) [inline docs/content/services/connect_mongodb_database_cluster.md:225]: - How can I **find** the connection details in the **Connect** tab?
- Where's the **Connection String Only** field?
- Mention the connection string in the documentation as well (you can provide the link to the section where I'm mentioning about retrieving the connection URI).
- It's better to point out the UI elements in the images which will be used by the user.

File: docs/content/services/connect_mongodb_database_cluster.md [hunk 2]
```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
...
+    <figure markdown>
+    <kbd>![compass-create-connection.png](../../img/services/connect_mongodb_database_cluster/compass-create-connection.png)</kbd>
+    </figure>
+
+    -   Then it will ask a name for the connection. You can give any name you want. Then click on **Save & Connect**.
+    <figure markdown>
+    <kbd>![compass-dave-connection.png](../../img/services/connect_mongodb_database_cluster/compass-save-connection.png)</kbd>
+    </figure>
+
+    -   When the connection is established, you can see the following screen. You can see the databases in your database cluster in the left side of the screen. You can see the collections in the selected database in the middle of the screen. You can see the data in the selected collection in the right side of the screen.
```
Comment (IsuruVihan) [inline docs/content/services/connect_mongodb_database_cluster.md]: - Present these information as an unordered list.
- Mark the relevant UI components in the image.

File: docs/content/services/connect_mongodb_database_cluster.md [hunk 3]
```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
...
+    <figure markdown>
+    <kbd>![compass-connection-landing.png](../../img/services/connect_mongodb_database_cluster/compass-connection-landing.png)</kbd>
+    </figure>
+
+    -   There is a tab called **Databases** in the top middle of the screen. You can click on it to view the databases in your database cluster. In here you can see some information about the databases. You can also create a new database by clicking on the **Create Database** button.
```
Comment (IsuruVihan) [inline docs/content/services/connect_mongodb_database_cluster.md:259]: - Do we need to mention about **Creating Database**?

File: docs/content/services/connect_mongodb_database_cluster.md [hunk 4]
```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
...
+    <figure markdown>
+    <kbd>![compass-databases.png](../../img/services/connect_mongodb_database_cluster/compass-databases.png)</kbd>
+    </figure>
+
+    -   When you open a database, you can see the collections in the database in the middle of the screen. You can also create a new collection by clicking on the **Create Collection** button.
+    <figure markdown>
+     <kbd>![compass-collections.png](../../img/services/connect_mongodb_database_cluster/compass-collections.png)</kbd>
```
Comment (IsuruVihan) [inline docs/content/services/connect_mongodb_database_cluster.md:276]: - Mark the relevant UI components in the image.

File: docs/content/services/connect_mongodb_database_cluster.md [hunk 5]
```diff
@@ -83,125 +81,177 @@ Now you are going to set up a MongoDB cluster to connect to your local applicati
...
+    </figure>
+
+    -   When you open a collection, you can see the data in the collection in the right side of the screen. You can also create a new document by clicking on the **Add data** button.
+    <figure markdown>
+    <kbd>![compass-view-documents.png](../../img/services/connect_mongodb_database_cluster/compass-view-documents.png)</kbd>
```
Comment (IsuruVihan) [inline docs/content/services/connect_mongodb_database_cluster.md:281]: - Mark the relevant UI components in the image.

**Review by IsuruVihan — CHANGES_REQUESTED** [review]
Do the changes that I've requested. After that I'll go through typo and comment on that as well.👌

File: docs/content/services/connect_mongodb_database_cluster.md (hunk 2 above)
Comment (IsuruVihan) [reply docs/content/services/connect_mongodb_database_cluster.md]: I can't see the list

Comment (hariwij) [reply docs/content/services/connect_mongodb_database_cluster.md]: image
Check this out. Changes are already done.

**Review by IsuruVihan — APPROVED** [review]
Looks good to me. Don't forget to check grammar twice before merging @hariwij

**Pull request merged**